hedyorg/hedy
# 💻 Go to next student button on reviewing

Adds two buttons, so we can decide which one to keep tomorrow in the meeting:

![image](https://github.com/user-attachments/assets/000e4f49-f83f-4465-a6e5-c3a5ffb7f146)

1. The "Go to next student" button, does what the original issue suggests, a button to review the next student that completed this adventure, i.e: iterating over columns.

2. The "Go next" button goes to the next adventure submitted by the student, i.e: iterating over rows.

I added both to see which one would you guys prefer.

Fixes #5772

**How to test**

*  Log in as teacher1 and go to the class.
* Click the eye icon in one of the programs.
* Check that when you click the Go next button you go to the next adventure
* Check that when you click go next student you go to the next student.
